### DISCUSSION TITLE
Using your files in a Project

### DISCUSSION THREAD

**Discussion by NikolaiVChr** [jazz-soft/JZZ-midi-GM#2] [Show and tell]
That's a very nice compilation of voices and kits.

Just want to notify you that I have taken them and will be using them in next release of Maestro MIDI to ABC converter.
It is also a MIT license so should be okay I hope. You will be credited in readme.md file.

I only had to change one of the drumkits, as they were starting from 1 instead of 0. Else the files are unchanged.

The link is here: https://github.com/NikolaiVChr/maestro

I hope you will approve of this use. :)

**Comment by jazz-soft** [comment]
Thanks for asking! Sure you can use the data!
The link to my repo is not mandatory, but I would really appreciate it.

> I only had to change one of the drumkits, as they were starting from 1 instead of 0.

Do you mean you found en error? Please point me to it.

:)

**Comment by NikolaiVChr** [comment]
Sorry, I was mistaken. I did that on the GS drum kits, and those I didn't get from you.
Nevermind that.

Glad you approve.
I have been editing a bit and expanding abbreviations in the lists. I can make a pull request soon, with those changes, then you can review and see if its changes you would like.

**Comment by jazz-soft** [comment]
Sounds great! Thanks a lot!
